project/ws/app/src/lib/routes/see-all/components/see-all-home/see-all-home.component.ts

```diff
@@ -157,7 +157,29 @@ export class SeeAllHomeComponent implements OnInit, OnDestroy {
   getInprogressAndCompleted(array: NsContent.IContent[], customFilter: any, strip: NsContentStripWithTabs.IContentStripUnit) {
     const inprogress: any[] = []
     const completed: any[] = []
-    array.forEach((e: any, idx: number, arr: any[]) => (customFilter(e, idx, arr) ? inprogress : completed).push(e))
+    array.forEach((e, idx, arr) => {
+      if (customFilter(e, idx, arr)) {
+        if ((e.status as string) !== 'Retired') {
+          inprogress.push(e)
+        }
+      } else {
+        completed.push(e)
+      }
+    })
+    // Sort the completed array with 'live' status first and 'Retired' status second
+    completed.sort((a: any, b: any) => {
+      if (a.status === 'live' && b.status !== 'live') {
+        return 1
+      }  if (a.status !== 'live' && b.status === 'live') {
```

Comment:
please check the status by converting it to lowercase

Reply:
converted plz check